spec/helpers/posts_helper_spec.rb

```diff
@@ -11,5 +11,5 @@
 #   end
 # end
 RSpec.describe PostsHelper, type: :helper do
-  pending "add some examples to (or delete) #{__FILE__}"
+  # pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
[OPTIONAL] I would suggest to make your code cleaner. If you need this test files kindly remove them.